pennylane/templates/embeddings/angle.py

```diff
@@ -67,11 +68,11 @@ def AngleEmbedding(features, wires, rotation='X'):
     ###############
 
     if rotation == 'X':
```

Comment:
One could possibly refactor this with a dict `{ 'X' : RX, ...}` to make it even more concise

Reply:
Let's leave this for a separate PR...